stablehlo/tests/infer_stablehlo.mlir

```diff
@@ -664,6 +664,19 @@ func.func @scatter_bounds(%input_tensor: tensor<200x?x?xf32, #stablehlo.bounds<?
 
 // -----
 
+// CHECK-LABEL: func @send
```

Comment:
Do we need a recv test?

Reply:
Type inference for recv is infeasible since we don't know what value we are "receiving"